htdocs/public/paypal/newpayment.php

```diff
@@ -872,14 +873,33 @@
 	print '<tr class="CTableRow'.($var?'1':'2').'"><td class="CTableRow'.($var?'1':'2').'">'.$langs->trans("Amount");
 	if (empty($amount))
 	{
-		print ' ('.$langs->trans("ToComplete");
-		if (! empty($conf->global->MEMBER_EXT_URL_SUBSCRIPTION_INFO)) print ' - <a href="'.$conf->global->MEMBER_EXT_URL_SUBSCRIPTION_INFO.'" rel="external" target="_blank">'.$langs->trans("SeeHere").'</a>';
-		print ')';
+	    print ' (';
+	    //check if user can change the amount ...
+	    if (! empty($conf->global->MEMBER_NEWFORM_EDITAMOUNT)) {
+	      print $langs->trans("ToComplete");
+	    }
+	    if (! empty($conf->global->MEMBER_EXT_URL_SUBSCRIPTION_INFO)) print ' - <a href="'.$conf->global->MEMBER_EXT_URL_SUBSCRIPTION_INFO.'" rel="external" target="_blank">'.$langs->trans("SeeHere").'</a>';
+	    print ')';
 	}
 	print '</td><td class="CTableRow'.($var?'1':'2').'">';
 	if (empty($amount) || ! is_numeric($amount))
 	{
 	    $valtoshow=GETPOST("newamount",'int');
+	    //check default subscription amount ...
+	    if ($conf->global->MEMBER_NEWFORM_EDITAMOUNT == true) {
```

Comment:
All constants/params in dolibarr setup are saved as a varchar in database. They have type string in php.
So 
 if ($conf->global->MEMBER_NEWFORM_EDITAMOUNT == true) {
should be replaced with 
 if (! empty($conf->global->MEMBER_NEWFORM_EDITAMOUNT)) {

So if constant is not defined or defined to 0, test is false.